netsim/modules/lag.py

```diff
@@ -327,16 +372,27 @@ def module_pre_transform(self, topology: Box) -> None:
   """
   def node_post_transform(self, node: Box, topology: Box) -> None:
     features = devices.get_device_features(node,topology.defaults)
+    has_peerlink = False
+    uses_mlag = False
     for i in node.interfaces:
-      if i.get('lag.mlag.peergroup',None):  # Fill in peer loopback IP and vMAC for MLAG peer links
+      if i.get(PEERLINK_ID_ATT,None):       # Fill in peer loopback IP and vMAC for MLAG peer links
         populate_mlag_peer(node,i,topology)
+        has_peerlink = True
       elif i.type=='lag':
         i.lag = node.get('lag',{}) + i.lag  # Merge node level settings with interface overrides
         lacp_mode = i.get('lag.lacp_mode')  # Inheritance copying is done elsewhere
         if lacp_mode=='passive' and not features.lag.get('passive',False):
           log.error(f'Node {node.name} does not support passive LACP configured on interface {i.ifname}',
             category=log.IncorrectAttr,
             module='lag',
-            hint='lag')
+            hint='passive LACP')
```

Comment:
The 'hint' attribute should refer to a key in "defaults.hints.module", otherwise it will not do anything.

Reply:
removed